example/lib/main.dart

```diff
@@ -246,6 +253,21 @@ class _MyAppState extends State<MyApp> {
               mainAxisAlignment: MainAxisAlignment.center,
               crossAxisAlignment: CrossAxisAlignment.center,
             ),
+            Container(
+              width: 56.0,
+              height: 56.0,
+              child: Slider(
+                value: slider_current_position,
+                min: 0.0,
+                max: max_duration,
+                onChanged: (double value) {
+                  setState(() async{
+                    String result = await flutterSound.seekToPlayer(value);
```

Comment:
Actually, you should've cast `double` to `int` conversion here since `seekToPlayer` receive `int` parameter.